Makefile

```diff
@@ -23,13 +23,18 @@
 # A simple check to determine the tool is available. No version check, etc.
 REQUIRED_SOFTWARE=docker docker-compose git node yarn
 
-# List of projects that will be cloned and integrated into the system.
-# The project name must be in the ReactionCommerce Github org and the name must
-# match the Github project name.
-# Projects are started in order of this list.
-REACTION_PROJECTS=reaction-hydra \
-		  reaction \
-		  reaction-next-starterkit
+# Defined here are the subprojects in a comma-separated format
+# URL_PREFIX,REPO_NAME,TAG
+# URL_PREFIX is the start of a git repository URL (no trailing slash)
+# REPO_NAME is just the directory name itself
+# TAG is the git tag or branch to checkout
+# Projects will be started in this order
+define reaction-repos
+git@github.com:/reactioncommerce,reaction-hydra.git,master \
+git@github.com:/reactioncommerce,reaction.git,master \
+git@github.com:/reactioncommerce,reaction-next-starterkit.git,master
```

Comment:
Using the git URL as the first arg is fine but we shouldn't that be a complete URL? I understand that we need another entry for the directory name but I think it would be better to duplicate rather than have two partial list items.

Reply:
OK made that change